Bing's space is not countably compact

Moniker1998 opened the pull request:
Since Micheal's closed subspace is not countably compact (if it were, it would be compact since countably compact + metacompact implies compact, but we know it's not paracompact), it follows that Bing's space is also not countably compact.

prabau commented on spaces/S000136/properties/P000019.md:
```diff
@@ -0,0 +1,10 @@
+  name: Counterexamples in Topology
+---
+
+{S000137} is a closed subspace of $X$, but {S000137} is not {P000019}
```
Even though it works either way, we like to use the shorter versions in the internal links, i.e., "{S137}" and "{P19}".  Makes it easier to work with when editing things.

Moniker1998 commented:
If we were to prove that Bing's space is realcompact (as in #373 ), then this theorem would probably be useless. I think I should change it to a draft for now?

Moniker1998 commented:
I think it'll be really hard to determine if Bing's example G is realcompact. So I'll just add the property of countable paracompactness I've seen along with some references and be at it

Moniker1998 commented:
I've decided to clean up S136 a little as well while I'm at it.

Properties I've added are
P8 (that is $T_5$), P19 (not countably compact), P32 (countably paracompact) and P139 (has isolated points)

I've deleted properties
P3, P7 (weaker separation axioms, pointless to have), P30 (it's not metacompact, so we don't need that it's not paracompact), P56 (it follows from having isolated points) and finally P129 because it follows from not being $\leq 2^\mathfrak{c}$ and being $T_5$.
